aqua-examples/ts-oracle/Readme.md

```diff
@@ -106,84 +161,140 @@ data Oracle:
     raw_data: []u64
 
 -- the point_estimate function from the Wasm file wrapped as a service
--- marine aqua artifacts/ts_oracle.wasm
+-- marine aqua ts-oracle/modules/ts-oracle/target/wasm32-wasi/release/ts_oracle.wasm
 service TSOracle("service-id"):
     point_estimate: []u64, u32 -> Oracle
 
 -- the timestamp getter drawing from the Kademlia neighborhood
 -- function collects the neighborhood peers
 -- retrieves and returns the timestamps 
-func ts_getter(node: string) -> []u64:
+-- the peer id gets from the wrapper of our service interface
+-- for handling timeouts from the neighborhood peers, the race pattern is used
+func ts_getter() -> []u64:
+  rtt = 1000 -- millis
+  msg = "timeout"
   res: *u64
-  on node:
-    k <- Op.string_to_b58(node)
+
+  services <- App.services()
+
+  on services.tsOracle.default!.peerId:
```

Comment:
if you're using `!` to access the first element in the array, you ought to say so.

Reply:
OK